src/components/Canvas/InteractionLayer.tsx

```diff
@@ -162,6 +208,14 @@ export default class InteractionLayer extends BaseLayer {
     return this.movingSelectedArea;
   }
 
+  getExtendingSelectedArea() {
+    return this.extendingSelectedArea;
+  }
+
+  getExtendingSelectedPixels() {
+    return this.extendingSelectedPixels;
+  }
```

Comment:
how about adding these to storybook example? let's show off great feature

Reply:
I will add an example for this in another PR later!